packages/utilities/__tests__/diff/DiffUtils.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import * as diff from "diff";
+import { DiffUtils } from "../../src/diff/DiffUtils";
+import { IDiffOptions } from "../../src/diff/doc/IDiffOptions";
+import { WebDiffManager } from "../../src/diff/WebDiffManager";
+import * as jestDiff from "jest-diff";
+import * as diff2html from "diff2html";
+jest.mock("diff");
+
+describe("DiffUtils", () => {
+
+    describe("getDiffString", () => {
+        const string1: string = "random string one";
+        const string2: string = "random string two";
+        const diffOptions: IDiffOptions = {
+            outputFormat: 'unifiedstring'
+        };
+
+        it("should return a diff string in unified string format", async () => {
+            jest.spyOn(jestDiff , "diff").mockImplementation(jest.fn());
+            expect(await DiffUtils.getDiffString(string1, string2, diffOptions)).toMatchSnapshot();
+        });
+        it("should return a diff string in format for terminal i.e, json diff", async () => {
+            diffOptions.outputFormat = "terminal";
+            const response = await DiffUtils.getDiffString(string1, string2, diffOptions)
```

Comment:
Please fix the lint warnings reported by GitHub Actions. You can also see them locally by running `npm run lint`.

Reply:
Rectified the errors. :slightly_smiling_face:

Reply:
seems to be addressed now